tests/test_python.py

```diff
@@ -216,10 +217,21 @@ def test_export_onnx():
 
 @pytest.mark.skipif(checks.IS_PYTHON_3_12, reason="OpenVINO not supported in Python 3.12")
 @pytest.mark.skipif(not TORCH_1_13, reason="OpenVINO requires torch>=1.13")
-def test_export_openvino():
+@pytest.mark.parametrize(
+    "dynamic, int8, half, batch, data, imgsz",
+    product([True, False], [False, True], [True, False], [1, 4], [None], [None, 160]),
```

Comment:
I'd reduce this batch [1 2] for speed and for imgsz 32 for speed also, or 64 if you run into issues at 32.